src/app/Models/Movie/Movie.php

```diff
@@ -15,5 +16,10 @@ protected static function newFactory(): MovieFactory
         return MovieFactory::new();
     }
 
+    public function genre()
+    {
+        return $this->belongsTo(Genre::class);
```

Comment:
Veza treba da bude many2many. Movie moze imati vise zanrova i zanr moze imati vise filmova
Sto znaci da ce ti trebati pivot tabela koja ce pokazati njihovu vezu mozes pogledati to malo

https://laravel.com/docs/9.x/eloquent-relationships#many-to-many